cmd/zed/zed.d/zed.rc

```diff
@@ -86,6 +86,14 @@
 #ZED_SPARE_ON_IO_ERRORS=1
 
 ##
+# Turn on/off enclosure LEDs when drives get DEGRADED/FAULTED.  This works for
+# device mapper and multipath devices as well.  Your enclosure must be
+# be supported by the Linux SES driver for this to work.
+#
+#ZED_USE_ENCLOSURE_LEDS=1
```

Comment:
If you don't have a enclosure, it's totally safe.  The `if [ ! -d /sys/class/enclosure ]` line will not see an enclosure, and so nothing happens.  If you do have an enclosure, you could possibly risk contention with other entities trying to control the LEDs (like ledmon?).